src/Svg/Surface/CPdf.php

```diff
@@ -436,9 +436,9 @@ public function __destruct()
     /**
      * Destination object, used to specify the location for the user to jump to, presently on opening
      *
-     * @param $id
-     * @param $action
-     * @param string $options
+     * @param string $id
```

Comment:
Hinting that $id is a string (here and in every other method) is incorrect. $id should be an integer representing the object number. That's why you're getting "X does not accept a string" warnings from phpstan.